docs/collecting-data/collecting-from-own-applications/python-tracker/emitters/index.md

```diff
@@ -250,7 +260,8 @@ e = AsyncEmitter("collector.example.com",
             byte_limit=None,
             max_retry_delay_seconds = 60,
             buffer_capacity = 5000,
-            event_store = None)
+            event_store = None,
+            session=None)
 ```
 
 ## The CeleryEmitter class
```

Comment:
Can you add a deprecation warning here and in the Redis emitter saying that they will be removed in v1?